cmd/captplanet/run.go

```diff
@@ -131,6 +134,15 @@ func cmdRun(cmd *cobra.Command, args []string) (err error) {
 		if runCfg.Satellite.Audit.SatelliteAddr == "" {
 			runCfg.Satellite.Audit.SatelliteAddr = runCfg.Satellite.Identity.Address
 		}
+
+		if runCfg.Satellite.Web.SatelliteAddr == "" {
+			runCfg.Satellite.Web.SatelliteAddr = runCfg.Satellite.Identity.Address
+		}
+
+		// Create satellite DB and pass it to web config
+		db, _ := satellitedb.New(runCfg.Satellite.Web.DatabaseDriver, runCfg.Satellite.Web.DataBaseURL)
```

Comment:
Don't ignore the error. Also, use the same field for setting both driver and databaseurl. `postgresql://host:23`, `sqlite://:fjklajsdlkf` etc. We have similar handling in other places.